static/app/components/errorRobot.spec.tsx

```diff
@@ -1,15 +1,14 @@
 import {render, screen, userEvent} from 'sentry-test/reactTestingLibrary';
 
-import {Client} from 'sentry/api';
 import ErrorRobot from 'sentry/components/errorRobot';
 
 describe('ErrorRobot', function () {
-  let getIssues;
+  let getIssues: jest.Func;
   let routerContext;
```

Comment:
is the type for `routerContext` correct or is it `any`? shouldn't we fix those too?

Reply:
I had a hard time figuring out what type to give this